File: sentimental-mario-less/mario.py
# TODO
import cs50
while True:
    try:
        height=cs50.get_int("Height: ")
        if height<9 and height>0:
            break
    except ValueError:
        print("That is not a integer!")
for i in range(0,height):
    for j in range(0,height-i-1):
        print(' ',end='')
    for h in range(0,i+1):
        print('#',end='')
    print("")

File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int Height;
    while(true){
        Height=get_int("Height: ");
        if(Height>0 && Height<9){
            break;
        }
    }
    for(int i=0;i<Height;i++){
        for(int j=0;j<Height-i-1;j++){
            printf(" ");
        }
        for(int h=0;h<i+1;h++){
            printf("#");
        }
        printf("\n");
    }
    return 0;
}